lib/irs_reporting/fraud_metrics_lg99_report.rb

```diff
@@ -0,0 +1,250 @@
+# frozen_string_literal: true
+
+require 'csv'
+begin
+  require 'reporting/cloudwatch_client'
+  require 'reporting/cloudwatch_query_quoting'
+  require 'reporting/command_line_options'
+rescue LoadError => e
+  warn 'could not load paths, try running with "bundle exec rails runner"'
+  raise e
+end
+
+module IrsReporting
+  class FraudMetricsLg99Report
+    include Reporting::CloudwatchQueryQuoting
+
+    attr_reader :time_range, :issuers
+
+    module Events
+      IDV_FINAL_RESOLUTION = 'IdV: Final Resolution'
+      SUSPENDED_USERS = 'User Suspension: Suspended'
+      REINSTATED_USERS = 'User Suspension: Reinstated'
+
+      def self.all_events
+        constants.map { |c| const_get(c) }
+      end
+    end
+
+    # @param [Array<String>] issuers
+    # @param [Range<Time>] time_range
+    def initialize(
+      issuers: nil,
+      time_range:,
+      verbose: false,
+      progress: false,
+      slice: 6.hours,
+      threads: 1
+    )
+      @issuers = Array(issuers).presence # always an Array or nil
+      @time_range = time_range
+      @verbose = verbose
+      @progress = progress
+      @slice = slice
+      @threads = threads
+    end
+
+    def verbose?
+      @verbose
+    end
+
+    def progress?
+      @progress
+    end
+    
+    def as_emailable_reports
+      [
+        Reporting::EmailableReport.new(
+          title: 'Definitions',
+          table: definitions_table,
+        ),
+        Reporting::EmailableReport.new(
+          title: 'Overview',
+          table: overview_table,
+        ),
+        Reporting::EmailableReport.new(
+          title: "Fraud Metrics",
+          table: fraud_metrics_table,
+          filename: 'fraud_metrics',
+        ),
+        Reporting::EmailableReport.new(
+          title: "Suspended User Metrics",
+          table: suspended_metrics_table,
+          filename: 'suspended_metrics',
+        ),
+        Reporting::EmailableReport.new(
+          title: "Reinstated User Metrics",
+          table: reinstated_metrics_table,
+          filename: 'reinstated_metrics',
+        ),
+      ]
+    end
+
+    def definitions_table
+      [
+        ['Metric', 'Unit', 'Definition'],
+        ['Fraud Rules Catch Rate', 'Count', 'The count of unique accounts flagged for fraud review.'],
+        ['Fraudulent credentials disabled', 'Count', 'The count of unique accounts suspended due to suspected fraudulent activity within the reporting month.'],
+        ['Fraudulent credentials reinstated', 'Count', 'The count of unique suspended accounts that are reinstated within the reporting month.'],
+      ]
+    end
+
+    def overview_table
+      [
+        ['Report Timeframe', "#{time_range.begin} to #{time_range.end}"],
+        # This needs to be Date.today so it works when run on the command line
+        ['Report Generated', Date.today.to_s], # rubocop:disable Rails/Date
```

Comment:
```suggestion
        ['Report Generated', Time.zone.today.to_s],
```

What doesn't work when it's run on the command line?  `Date.today` isn't aware of Rails time zones and should be avoided if possible.